src/reducers/flexColumnWidths.js

```diff
@@ -0,0 +1,94 @@
+/**
```

Comment:
I added this file to solely take care of filling up flex widths for columns.
(flex is a column prop that allows the column to grow up to fill up the container)

This was earlier done by the `columnWidths` selector which was pretty bloated in my opinion because it took care of ordering columns, ordering column groups, and fixing up flex widths.
Also it was a selector so there was no way to update our prefix interval tree and other objects in our store's "internal" state